src/app/Main/Header/Export/Header-ExportContainer.component.tsx

```diff
@@ -61,7 +61,10 @@ export function HeaderExportContainerComponent(): JSX.Element {
         setting.tableAttributeNames.column = undefined
     }
 
-    const isolateFilteredUrl: string = ISOLATE_URL + history.location.search;
+    const filterSettingsPath = history.location.search.replace(/&row.+/g, "").replace(/&column.+/g, "")

```

Comment:
As stated in the grooming, just deleting here is not the best practice. E.g. if you happen to add another client side param you would have to delete it here to. Better is to pick the things you need here. You could try to use: ``history.location.state.params`` to get the params for your isolates query.

Reply:
`history.location.state.params` is no valid command. Of cause I could extract all parameter of the url (which would include row and col as well). But then I would have to extract the filter parameters and reassemble them into a string. That seems like an overkill and I think I'll leave it as it is. At the moment, no further parameters for the URL are planned.

Reply:
you can try the hook `useLocation` and use `location.state.params` and check if this works. If not you can still use url parsing: https://developer.mozilla.org/en-US/docs/Web/API/URL/URL to get the params.
If you want to leave it as it is, then you should at least ensure that only the desired part of the url is removed. Your regexp patterns just remove all things after `&row`. But url params can have any order and a user can change it and the client should still process it as a valid request. So even here you should use a the recommended utilities to do the work for you.